### DISCUSSION TITLE
Is there a way to communicate with local RWKV?

### DISCUSSION THREAD

**Discussion by 123385918** [langflow-ai/langflow#440] [Ideas]
As I know, langflow can't use RWKV to handle tasks. Is there an easy way to do this?

**Comment by Cristhianzl** [comment]
Hi! 👋

We are using other channels as our official means of communication with users. We apologize for the delayed response.
For faster assistance, please direct your questions to our Discord channel or open a GitHub issue, as we are much more active on these platforms.

[Langflow Discord Server](https://discord.com/invite/EqksyE2EX9)

Thank you for your understanding.

Best regards,
The Langflow Team
